src/galaxy/CustomSystem.cpp

```diff
@@ -571,10 +587,12 @@ static int l_csys_add_to_sector(lua_State *L)
 	(*csptr)->sectorX = x;
 	(*csptr)->sectorY = y;
 	(*csptr)->sectorZ = z;
-	(*csptr)->pos = vector3f(*v);
+	(*csptr)->pos = vector3f(*v) * Sector::SIZE; // NOTE: lua uses 0..1 interval inside a sector cell
```

Comment:
The raw value used by the entire codebase for system positions within a sector is in lightyears in the range [0..8) ly as sector sizes are defined. It's much easier and more intuitive for the user to edit the sector position in physical units, rather than an abstract scalar of 0..1 which requires an *additional* hand-calculation by the user to place the system in the sector. This multiplication was moved here only to maintain backwards compatibility with existing lua files.

Reply:
On the other hand, you need to know exactly the sector size to understand where the system is located. _Personally_, coordinate 0..1 seems to me more intuitive, in addition, it is consistent with the coordinates of the sector - where the integer just means 1 sector - 8 light years. But this doesn’t make any fundamental difference to me.